databallpy/load_data/tracking_data/_add_player_data_to_dict.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+
+def _add_player_data_to_dict(team_id:str, shirt_num:str, x:str, y:str, data: dict, idx: int) -> dict:
```

Comment:
Maybe make clear this is tracking data data, not metadata data `_add_player_tracking_data_to_dict()`